src/EventStore.Core/Util/Opts.cs

```diff
@@ -431,5 +431,10 @@ public static class Opts {
 
 		public const string WriteStatsToDbDescr = "Set this option to write statistics to the database.";
 		public const bool WriteStatsToDbDefault = true;
+
+		public static readonly long MaxTruncationDefault = -1;
```

Comment:
```suggestion
		public static readonly long MaxTruncationDefault = 256 * 1024 * 1024;
```
A single chunk is probably a sensible default here, and could prevent others from only learning about this setting after they've run into this problem and had their data truncated